### PR TITLE
add more usage info for kubectl wait option --for

### PR DESCRIPTION
**Pull request by pandaamanda**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

When use command kubectl wait -h, user can known  that condition value is allowed to specified.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go [hunk 1]
```diff
@@ -134,7 +134,7 @@ func (flags *WaitFlags) AddFlags(cmd *cobra.Command) {
 	flags.ResourceBuilderFlags.AddFlags(cmd.Flags())
 
 	cmd.Flags().DurationVar(&flags.Timeout, "timeout", flags.Timeout, "The length of time to wait before giving up.  Zero means check once and don't wait, negative means wait for a week.")
-	cmd.Flags().StringVar(&flags.ForCondition, "for", flags.ForCondition, "The condition to wait on: [delete|condition=condition-name].")
+	cmd.Flags().StringVar(&flags.ForCondition, "for", flags.ForCondition, "The condition to wait on: [delete|condition=condition-name]. The default status value of condition-name is true, you can set false with condition=condition-name=false")
```
Comment (dougsland) [inline staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go:137]: As we are touching in these bits. Can we add example as well?

"The condition to wait on: [delete|condition=condition-name]. For example, 'kubectl wait --for=condition=ready pod/mypodName --timeout=30s'. This command will wait the pod "mypodName" get ready for 30 seconds. The default status value of condition-name is true, you can set false with condition=condition-name=false"

Comment (pandaamanda) [reply staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go:137]: sure.

**Review by dougsland — COMMENTED** [review]
/lgtm

**Pull request merged**